substrate/frame/multisig/src/lib.rs

```diff
@@ -558,8 +587,8 @@ impl<T: Config> Pallet<T> {
 
 				// Clean up storage before executing call to avoid an possibility of reentrancy
 				// attack.
-				<Multisigs<T>>::remove(&id, call_hash);
-				T::Currency::unreserve(&m.depositor, m.deposit);
+				<MultisigsFor<T>>::remove(&id, call_hash);
+				let _ = m.ticket.drop(&m.depositor);
```

Comment:
Does it panic when its not manually dropped?

Reply:
You mean if someone forgot to call `drop` on the ticket?

Reply:
in general i think `let _` should be a no-no across the repo.

If in this situation, you specifically want to drop the result you should call `mem::drop` (AFAIK)